src/app/profile/profile.page.ts

```diff
@@ -0,0 +1,58 @@
+import { Component} from '@angular/core';
+import { ProfileService } from '../services/profile/profile.service';
+import { NavController } from '@ionic/angular';
+import { LoaderService } from '../services/loader/loader.service';
+
+@Component({
+  selector: 'app-profile',
+  templateUrl: './profile.page.html',
+  styleUrls: ['./profile.page.scss'],
+})
+export class ProfilePage {
+  formJson: any = [];
+  formData: any;
+  
+  constructor(private profileService: ProfileService,
+    private navCtrl: NavController,
+    private loader: LoaderService,
+  ){}
+
+  ionViewWillEnter() {
+    this.loadFormAndData();
+  }
+
+  async loadFormAndData() {
+    await this.loader.showLoading("Please wait while loading...");
+
+    this.profileService.getFormJsonAndData().subscribe(([formJsonRes, profileFormDataRes]: any) => {
+      if (formJsonRes?.status === 200) {
+        this.formJson = formJsonRes?.result?.data;
+      }
```

Comment:
same suggestion as previous one, see if we can combine the conditons into a single and handle the else case